src/functions/rule.js

```diff
@@ -20,8 +20,8 @@ defineFunction({
         return {
             type: "rule",
             shift: shift && shift.value,
-            width: width.value,
-            height: height.value,
+            width: width.value.value,
+            height: height.value.value,
```

Comment:
@kevinbarabash Sorry I missed this, but this is actually incorrect!
In replacing the use of the `ParseNode<*>` type with (the actually correct) `AnyParseNode`, the compiler caught this!
PR will come soon-ish...with various flow type fixes.

Reply:
Whoops.  Thanks for spotting that.

Reply:
And actually, I confused myself. While these two lines were correct, shift.value should also have had a `.value` added on to it.